shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/viewModel/fetchApi.kt

```diff
@@ -0,0 +1,34 @@
+package com.mbta.tid.mbta_app.viewModel
```

Comment:
suggestion (non-blocking): this, getGlobalData, getScheduls, and subscribeToPredictions don't totally feel like they fit into the viewModel package to me. Maybe they could go into a separate package along the lines of `apiRequests`, `getData`, etc.

Reply:
Since they’re Composables, the only shared code that can use them is the ViewModels, but a subpackage would be reasonable.